src/geneweaver/client/gedb.py

```diff
@@ -134,23 +183,117 @@ def sort(
         """
         ret = {}
         for dr in expressions:
-            strain = dr.get(prop)
+            pvalue = dr.get(prop)
 
-            collection = ret.get(strain, None)
+            collection = ret.get(pvalue, None)
             if collection is None:
                 collection = []
-                ret[strain] = collection
+                ret[pvalue] = collection
             collection.append(dr)
 
         return ret
 
+    def sort_for_concordance(
+        self, prop: str, expressions: List[DataResult]
+    ) -> Mapping[str, List]:
+        """Sort the data results by property then group the genes and individual names.
+
+        @param property: String e.g. "strain" to sort by strain
+        @param the raw list of data results returned from a 'search' call.
+        """
+        ret = {}
```

Comment:
In case you're expecting these values to be in any particular order `{}` makes unsorted dictionaries, but if you're just grouping by then it's all good